lib/resolveLang.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var defaultLangCode = require('./defaultLangCode'),
+	normalizeLangCode = require('./normalizeLangCode');
+
+var locales = [
```

Comment:
I thought the locale was comprised of (at least) language and region.  Should this data be referred to as "languages", and the next array be referred to as "locales"?